clouds/aws/aws_roles/aws_roles.py

```diff
@@ -0,0 +1,70 @@
+import botocore.errorfactory
+from aws.aws_clients import AWSClient
+from botocore.exceptions import BotoCoreError
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+def get_iam_client():
+    return AWSClient(service_name="iam").client
+
+
+def create_role_policy(
+    role_name, policy_name, policy_document_path, apply_update=False
+):
+    """
+    Create the role policy with specific configuration.
+
+    Args:
+        role_name (str): role policy name
+        policy_name (str): policy name
+        policy_document_path (str): path to Json file that holds the policy documents
+        apply_update (bool): True to update the role policy if found, else False
+
+    Raises:
+        OSError: If file read failed
+        BotoCoreError: If POST request failed
+    """
+    client = get_iam_client()
+    LOGGER.info(f"Creating new role {role_name} for policy {policy_name}.")
+    if apply_update or not check_if_role_policy_exists(
```

Comment:
I don't think `apply_update` needed, if the user call this function and pass role he want to create/update it.

Reply:
So we want to delete [check_if_role_policy_exists](https://github.com/RedHatQE/cloud-tools/pull/14/files#diff-773825122a0092f2831ee9d5485dc08a8701fe6a32a2ea7ad33c0dfb2cbbb95dR53)? If that is the work assumption, I don't see a reason to have it.

Reply:
Updated  in recent commit